src/Avalonia.Controls/Application.cs

```diff
@@ -68,6 +68,11 @@ public class Application : AvaloniaObject, IDataContextProvider, IGlobalDataTemp
         /// <inheritdoc/>
         public event EventHandler? ActualThemeVariantChanged;
 
+        static Application()
+        {
+            RequestedThemeVariantProperty.OverrideDefaultValue<Application>(ThemeVariant.Default);
```

Comment:
nit: RequestedThemeVariantProperty should already have `Default` as a default value.

Reply:
Currently, it doesn't. AddOwner doesn't inherit the default value of the property.